Prac_06/languages.py

```diff
@@ -0,0 +1,14 @@
+from Prac_06.programming_language import ProgrammingLanguage
+
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+
+programs = [ruby, python, visual_basic]
+
+print(ruby)
+
+print("The dynamically typed languages are:")
+for program in programs:
+    if program.is_dynamic() == True:
```

Comment:
`if program.is_dynamic() == True:` can also be written as: `if program.is_dynamic():`